templates/zerver/api/arguments.json

```diff
@@ -278,5 +278,13 @@
             "required":"Optional",
             "example":"['ZOE@zulip.com']"
         }
+    ],
+    "upload-image.md":[
+        {
+            "argument":"file",
+            "description": "File to be uploaded.",
+            "required":"Required",
+            "example":"See code examples above."
```

Comment:
We seem to have inconsistent whitespace formatting for these JSON files, I assume because they're hand-written?  Might want to do some linting; personally, I'd want there to always be a ` ` after a `:`.